### PR TITLE
chore(ci): bench multiple scalar bit sizes and add operators

### PR DESCRIPTION
**Pull request by soonum**
Scalar operations benchmark is done against scalar bit sizes.
Scalar division and modulo are added to the benchmark set.

### CODE REVIEW

**Comment by soonum** [comment]
@slab-ci integer_bench

**Comment by soonum** [comment]
@slab-ci integer_bench

**Comment by IceTDrinker** [comment]
potential issues : the parse integer to csv would have a broken format in that case (not hard to update) but would also need to handle scalar formatted benches (to get the scalar bit size) and the non scalar variant, I have an updated piece of code somewhere but I think it begs the question of the format in general, do we need the ServerKey part for example ? as it's common between shortint and integer I think I would replace it by the module name so something like shortint::add_parallelized instead of ServerKey::add_parallelized or similar

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci shortint_full_bench

**Comment by soonum** [comment]
@slab-ci integer_full_bench

**Comment by soonum** [comment]
@slab-ci shortint_full_bench

**Comment by soonum** [comment]
@slab-ci shortint_full_bench

File: .github/workflows/integer_full_benchmark.yml [hunk 1]
```diff
@@ -41,7 +46,7 @@ jobs:
       - name: Checkout tfhe-rs repo with tags
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
         with:
-          fetch-depth: 0
+            fetch-depth: 0
```
Comment (IceTDrinker) [inline .github/workflows/integer_full_benchmark.yml:49]: indent looks wrong

**Review by IceTDrinker — COMMENTED** [review]
Just a nitpick and then go

**Review by IceTDrinker — APPROVED** [review]

**Comment by IceTDrinker** [comment]
Thanks !

**Pull request merged**